lms/djangoapps/discussion/static/discussion/js/spec/discussion_board_factory_spec.js

```diff
@@ -0,0 +1,57 @@
+define(
+    [
+        'jquery',
+        'backbone',
+        'common/js/spec_helpers/page_helpers',
+        'common/js/spec_helpers/discussion_spec_helper',
+        'discussion/js/discussion_board_factory'
+    ],
+    function($, Backbone, PageHelpers, DiscussionSpecHelper, DiscussionBoardFactory) {
+        'use strict';
+
+        // TODO: re-enable when this doesn't interact badly with other history tests
```

Comment:
@bjacobel I gave up on getting this test to work. It passes when run individually, but when run in the full suite it causes other router-based tests to fail. It's your call as to whether you want to try and enable this again and work through the issues.